modules/gapi/src/backends/ocl/goclcore.cpp

```diff
@@ -523,6 +541,47 @@ GAPI_OCL_KERNEL(GOCLTranspose, cv::gapi::core::GTranspose)
     }
 };
 
+GAPI_OCL_KERNEL(GOCLBGR, cv::gapi::streaming::GBGR)
+{
+    static void run(const cv::MediaFrame& in, cv::UMat& out)
+    {
+        (void)in; (void)out;
+#ifdef HAVE_DIRECTX
+#ifdef HAVE_D3D11
+#ifdef HAVE_ONEVPL
+        auto d = in.desc();
+        GAPI_Assert(d.fmt == cv::MediaFormat::NV12);
```

Comment:
for troubleshooting purposes it had better make explicit check and print `LOG_ERROR("... unsupported, requested format: ", d.fmt)` and thow error then

Reply:
Done